packages/api/src/main/java/com/osahft/api/service/TransferService.java

```diff
@@ -19,10 +19,14 @@
 import java.util.List;
 import java.util.stream.Collectors;
 
+import static com.osahft.api.document.MailTransfer.State.*;
+
 @Service
 @Slf4j
 public class TransferService implements TransferServiceIF {
 
+    private static final int NUMBER_OF_AUTHENTICATION_ATTEMPTS = 5;
```

Comment:
maybe lower number of attempts to 3 like any other pin/password-like input limit?

Reply:
will adapt :)